modules/ossm-deploying-jaeger.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/customizing-installation-ossm.adoc
+
+
+[id="ossm-deploying-jaeger_{context}"]
+= Deploying Jaeger
+
+The Jaeger Operator currently supports the following deployment strategies:
+
+* *allInOne* (Default) - This strategy is intended for development, testing, and demo purposes. The main backend components, Agent, Collector and Query service, are all packaged into a single executable which is configured (by default) to use in-memory storage.
++
+[NOTE]
+====
+In-memory storage is not persistent, which means that if the Jaeger instance shuts down, restarts, or is replaced, that your trace data will be lost.  And in-memory storage cannot be scaled, since each pod has its own memory. For persistent storage, you must use the `production` or `streaming` strategies, which use Elasticsearch as the default storage.
+====
+
+* *production* - The production strategy is intended for production environments, where long term storage of trace data is important, as well as a more scalable and highly available architecture is required. Each of the backend components is therefore deployed separately.  The Agent can be injected as a sidecar on the instrumented application or as a daemonset. The Query and Collector services are configured with a supported storage type - currently Elasticsearch. Multiple instances of each of these components can be provisioned as required for performance and resilience purposes.
```

Comment:
Agent daemonset is not currently supported (atleast in the Service Mesh context) as it would break multitenancy.

Reply:
I'm trying to use this topic for both sets of docs.  Let me think about how to note that daemonset is standalone only.

Reply:
I don't know that we have agreed it will be supported in standalone - the problem is if in a cluster that also has service mesh installed, then could cause issues if the daemonset is on the same nodes as service mesh.
@rcernich could you clarify the service mesh position on this?

Reply:
I think the issues had more to do with service mesh installing jaeger this way, with main issue being you could only have a single daemon set install per cluster.  That said, we've not done any testing around integrating with jaeger running as a daemon set, so I don't know if you could connect service mesh to it or not.

Reply:
Ok, @JStickler can you remove the reference to daemonset for now until it has been tested and shown not to interfere with service mesh.

Reply:
Removed daemonset references.